src/cascadia/TerminalSettingsEditor/Profiles.xaml

```diff
@@ -144,81 +144,77 @@
+                                                             Color="{ThemeResource SystemColorHighlightColor}" />
+                                            <SolidColorBrush x:Key="ButtonForeground"
+                                                             Color="{ThemeResource SystemColorButtonTextColor}" />
+                                            <SolidColorBrush x:Key="ButtonForegroundPointerOver"
+                                                             Color="{ThemeResource SystemColorHighlightTextColor}" />
+                                            <SolidColorBrush x:Key="ButtonForegroundPressed"
+                                                             Color="{ThemeResource SystemColorHighlightTextColor}" />
+                                        </ResourceDictionary>
+                                    </ResourceDictionary.ThemeDictionaries>
+                                </ResourceDictionary>
+                            </Button.Resources>
+                            <Button.Content>
+                                <StackPanel Orientation="Horizontal">
+                                    <FontIcon FontSize="{StaticResource StandardIconSize}"
+                                              Glyph="&#xE74D;" />
+                                    <TextBlock x:Uid="Profile_DeleteButton"
+                                               Margin="10,0,0,0" />
+                                </StackPanel>
+                            </Button.Content>
+                            <Button.Flyout>
+                                <Flyout>
+                                    <StackPanel>
+                                        <TextBlock x:Uid="Profile_DeleteConfirmationMessage"
+                                                   Style="{StaticResource CustomFlyoutTextStyle}" />
+                                        <Button x:Uid="Profile_DeleteConfirmationButton"
+                                                Click="DeleteConfirmation_Click" />
                                     </StackPanel>
-                                </Button.Content>
-                                <Button.Flyout>
-                                    <Flyout>
-                                        <StackPanel>
-                                            <TextBlock x:Uid="Profile_DeleteConfirmationMessage"
-                                                       Style="{StaticResource CustomFlyoutTextStyle}" />
-                                            <Button x:Uid="Profile_DeleteConfirmationButton"
-                                                    Click="DeleteConfirmation_Click" />
-                                        </StackPanel>
-                                    </Flyout>
-                                </Button.Flyout>
-                            </Button>
-                            <TextBlock x:Name="DeleteButtonDisclaimer"

```

Comment:
We can really get rid of this disclaimer? I was under the impression that deleting a inbox profile from your `settings.json` would still have it be present in the list of profiles, since those are in `defaults.json` (they're not from a dynamic source). 

Or do we log the default profile guids in `state.json` somewhere? (I may have missed this)

Reply:
Even default profiles can be removed now. If a profile isn't present in the user's settings.json, it's origin isn't `OriginTag::User`. In such cases the profile will now always be marked as "hidden" (and "deleted") unless it's the first time we're encountering it.

This works even for inbox profiles.

Reply:
Oh and for your last question specifically: Yeah! We're logging all GUIDs to state.json.

Reply:
oh yea okay, I see it now with the `else if (profile.Origin() != OriginTag::User)` okay that makes sense!